src/components/Nav/index.js

```diff
@@ -188,10 +184,74 @@ const Nav = ({ handleThemeChange, isDarkTheme, path }) => {
       to: `/learn/`,
       shouldDisplay: contentVersion < 1.1,
     },
+    {
+      text: "page-home-section-individuals-item-two",
+      ariaLabel: "nav-use-aria-label",
+      shouldDisplay: contentVersion > 1.1,
+      items: [
+        {
+          text: "page-home-section-individuals-item-one",
+          to: "/what-is-ethereum/",
+          shouldDisplay: contentVersion > 1,
+        },
+        {
+          text: "page-home-section-individuals-item-four",
+          to: "/eth/",
+          shouldDisplay: contentVersion > 1,
+        },
+        {
+          text: "page-home-section-individuals-item-six",
+          to: "/get-eth/",
+          shouldDisplay: contentVersion > 1.1,
+        },
+        {
+          text: "page-find-wallet-explore-dapps",
+          to: "/dapps/",
+          shouldDisplay: contentVersion > 1,
+        },
+        {
+          text: "page-home-section-individuals-item-five",
+          to: "/wallets/",
+          shouldDisplay: contentVersion > 1,
+        },
+      ],
+    },
+    {
+      text: "page-home-section-learn-title",
+      ariaLabel: "nav-learn-aria-label",
+      shouldDisplay: contentVersion > 1.1,
+      items: [
```

Comment:
@ryancreatescopy what's your thoughts on the ordering of these links? My 2 gwei: I think "Guides and resources" should probably be at/near the top given that it's more intro-level vs. EIPs.

Reply:
(I agree... Think it belong first, then whitepaper which started it all, then EIPs which evolved it, then move on to Eth2, and cap it with the glossary for reference... just my two wei... and note my two wei are 10^-9 as valuable as sam's 2 gwei hah)

Reply:
I'm happy with that ordering @wackerow – EIPs shouldn't be at the top, that's for sure.